iocore/net/quic/test/test_QUICFlowController.cc

```diff
@@ -62,6 +77,7 @@ TEST_CASE("QUICFlowController_Local_Connection", "[quic]")
   CHECK(fc.current_offset() == 1024);
   CHECK(fc.current_limit() == 1024);
   CHECK(ret == 0);
+  Thread::get_hrtime_updated();
```

Comment:
What is this for?

Reply:
We need to update the hrtime. Otherwise the value returned from `Thread::get_hrtime` is always the same one. It may cause the calculation of rate failed.

Reply:
https://github.com/scw00/trafficserver/blob/75d5062dc20280afff36695b4c9ce0c1e5fabdd5/iocore/net/quic/QUICFlowController.cc#L31-L37